blivetgui/osinstall.py

```diff
@@ -132,6 +132,12 @@ def __init__(self, client, spoke, spoke_container):
         self.physical_view = PhysicalView(self)
         self.builder.get_object("scrolledwindow_physical").add(self.physical_view.vbox)
 
+    def ui_refresh(self, _spoke):
+        # some configuration for ui elements needs to be done when the ui
+        # is actually visible -- in Anaconda we need to do this using the
+        # spoke "entered" signal
+        self.list_devices.disks_view.set_cursor(0)
```

Comment:
So this is hooked/called in/from Anaconda? I think this comment should be a proper docstring explaining what the method does and when it should be called.